modules/op-release-notes-1-11.adoc

```diff
@@ -275,3 +275,13 @@ If you installed the `tkn` command line tool using RPM on {op-system-base-full},
 == Known issues
 
 * If you use the Bundles resolver to define a pipeline run and then use the `tkn pac resolve --v1beta1` command for this pipeline run, the command generates incorrect YAML output. The `kind` parameter for the bundle is set to `Task` in the YAML output. As a workaround, you can set the correct value in the YAML data manually. Alternatively, you can use the `opc pac resolve --v1beta1` command or use the version of the `tkn` tool included with {pipelines-shortname} version 1.12.0 or later.
+
+[id="release-notes-1-11-3_{context}"]
+== Release notes for {pipelines-title} General Availability 1.11.3
+
+With this update, {pipelines-title} General Availability (GA) 1.11.3 is available on {OCP} 4.12 and later versions.
+
+[id="fixed-issues-1-11-3_{context}"]
+=== Fixed issues
+
+* Before this update, if the final task of a pipeline has failed or wws skipped, {pipelines-shortname} reported validation errors. With this update, a pipeline can succeed even if its final task fails or is skipped.
```

Comment:
@mramendi typo,
* wws skipped => was skipped

Reply:
IMHO the issue description is not accurate. It was failing when pipeline was re-publishing a result of a failed/skipped task.

reproducer in https://issues.redhat.com/browse/SRVKP-3975?focusedId=23797650&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-23797650

Reply:
Actually, I was wrong, ignore previous comment.